docs/sources/alerting/set-up/configure-alertmanager/index.md

```diff
@@ -16,6 +16,9 @@ weight: 200
 
 Set up Grafana to use an external Alertmanager as a single Alertmanager to receive all of your alerts. This external Alertmanager can then be configured and administered from within Grafana itself.
 
+**Note:**
```

Comment:
I think we should use the adminition instead, it automatically adds **Note:** and wraps it in a quote.

```
{{% admonition type="note" %}}
Grafana Alerting does not support sending alerts to the AWS Managed Service for Prometheus due to the lack of sivv4 support in Prometheus.
{{% /admonition %}}
```

Reply:
Interesting, it was not used for the other note in that page.

Reply:
Yea I think the admonition was introduced perhaps after that note was written? I don't think all existing ones were updated when it was added :D